azure-pipelines.yml

```diff
@@ -238,9 +238,8 @@ jobs:
       export RISCV_TARGET=opentitan
       export OT_TARGET=verilator
       export OT_TOOLS="$TOOLCHAIN_PATH/bin"
-      cd sw/vendor/riscv_compliance
       # Only running base interger variant for now, others will be added later.
-      make RISCV_ISA=rv32i
+      make -C sw/vendor/riscv_compliace RISCV_ISA=rv32i
```

Comment:
i actually originally had this in the other format, and I believe @imphil asked me to change to this. I can't really remember why now...

Reply:
Done.